src/py/aspen/covidhub_import/implementation.py

```diff
@@ -48,6 +55,13 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class Auth0Entry:
+    nickname: str
```

Comment:
we don't need to add username to this? or does nickname mean username?

Reply:
For the covidtracker auth0 account, the usernames were all email addresses.

Reply:
oh ok then nevermind